### PR TITLE
MCOL-1107 basic Java implementation of cpimport

### PR DESCRIPTION
**Pull request by sneJ-**

### CODE REVIEW

File: example/CpImport/src/main/java/com/mariadb/columnstore/api/examples/CpImport.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+	 * Parses the date to a MariaDB supported format.
+	 * 
+	 * @param date,
+	 *            input to import
+	 * @param dateFormat,
+	 *            of input in SimpleDateFormat notation
+	 * @return parsed date in MariaDB format, or null in case of error.
+	 */
+	private String parseDate(String date, String dateFormat) {
+		SimpleDateFormat inputParser = new SimpleDateFormat(dateFormat);
```
Comment (davidwbt) [inline example/CpImport/src/main/java/com/mariadb/columnstore/api/examples/CpImport.java]: I believe it's still true that SimpleDateFormat is still considered somewhat expensive to construct each time so moving it to a class member variable and reusing might be faster. Same for method below.

File: example/CpImport/src/test/java/com/mariadb/columnstore/api/examples/CpImportTest.java [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+	 */
+	@Test
+	public void testBasicWithParsing() {
+		// create test table
+		Connection conn = getConnection();
+		String TABLE_NAME = "t2";
+		String DELIMITER = "\\|";
+		executeStmt(conn, "CREATE TABLE IF NOT EXISTS " + TABLE_NAME + "(d date, dt datetime) ENGINE=columnstore");
+
+		// simple insert test without parsing
```
Comment (davidwbt) [inline example/CpImport/src/test/java/com/mariadb/columnstore/api/examples/CpImportTest.java]: Maybe add a test where you have a empty value for date and datetime?
